core/domain/user_jobs_continuous_test.py

```diff
@@ -259,7 +259,12 @@ def test_basic_computation_works_if_exploration_is_deleted(self):
                 EXP_ID, USER_ID, title=EXP_TITLE, category='Category')
             last_updated_ms_before_deletion = (
                 self._get_most_recent_exp_snapshot_created_on_ms(EXP_ID))
-            exp_services.delete_exploration(USER_ID, EXP_ID)
+            # Set remove_from_subscribers to False because we depend on
+            # RecentUpdatesMRJobManager to loop over UserSubscriptionsModel for
+            # explorations the user has subscribed to and check changes.
+            exp_services.delete_exploration(
+                USER_ID, EXP_ID,
+                remove_from_subscribers=False)
```

Comment:
I think we shouldn't have a third parameter here. Can't we update the behaviour to not loop over explorations that are deleted?

Reply:
I see, I guess we could remove the unit tests related to deleted explorations in user continuous jobs.

Reply:
Done.